config.json

```diff
@@ -118,7 +118,9 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "strings",
+        "equality",
+        "control-flow (if-else statements)"
```

Comment:
Please change it to `conditionals` for consistency with https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt